js/components/number.js

```diff
@@ -14,7 +14,20 @@ Fliplet.FormBuilder.field('number', {
       default: 0
     }
   },
+  methods: {
+    updateValue: function () {
+      var ensureNumberRx = new RegExp(
+        this.positiveOnly ? '[^0-9\.,]' : '[^0-9\.,-]'
+      , 'g');
+
+      this.value = this.value.replace(ensureNumberRx, '');
+      this.$emit('_input', this.name, this.value);
```

Comment:
So, anything that isn't in the pattern `[^0-9\.,]` or `[^0-9\.,-]` are excluded. That's good.

When the data is submitted to the server, will the `,` be removed so that the `,` is only used for display purposes?

Having `,` saved in the data sources could create problems if there aren't currently sufficient type checking/casting in place.

Reply:
@tonytlwu good point, should I just `.replace(/,/g, '.')` here on line 23?

Reply:
I imagine it'll be `.replace(/,/g, '')` based on UK/US standards, but the challenge, as your example reminds me, is that `.replace(/.//g, '')` is what should be happening for EU users.

Not sure how to best resolve this.

Reply:
well, this replacement happens on each keypress, meaning that when you type a comma it gets replaced to a dot instantly. That would should be quite obvious what the meaning is? What do you think?

Also: keep in mind we currently don't do any replacement so it's potentially a breaking change for companies like Socotec which are managing this on their own.

Reply:
If when a user pastes `12,300,230.00` it becomes `12.300.230.00` I think:

1. It's a little confusing. User can stare at it and work it out after a while, but...
1. User could end up just submitting it as it is as there aren't any more sophisticated validations in place

My lazy brain tells me `.replace(/,/g, '')` should do the trick. `parseFloat('12,300')` currently gives me `12` so we should probably do that anyway.

Reply:
@tonytlwu I've basically updated the initial regexp to not include `,` :-)

Reply:
@tonytlwu and it works great 

![gif](https://cl.ly/a28347fac563/Screen%252520Recording%2525202019-05-22%252520at%25252002.12%252520PM.gif)